macOS/DuckDuckGo/Feedback/View/FeedbackViewController.swift

```diff
@@ -73,6 +73,8 @@ final class FeedbackViewController: NSViewController {
     @IBOutlet weak var requestFeatureItem: NSMenuItem!
     @IBOutlet weak var reportProblemITem: NSMenuItem!
 
+    private let supportedOSChecker = SupportedOSChecker()
```

Comment:
I know this should ideally be injected, but I draw the line here as to how much I want to improve the code.  Having this here should allow us to inject it rather easily if required in the future.